vocab/credentials/v2/issuer-dependent.html

```diff
@@ -66,14 +68,38 @@
     <section>
         <h2>Issuer Dependent Terms</h2>
         <p>
-            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> of a Verifiable Credential, as defined in [vc-data-model], provides a mapping from the standard terms used in credentials to the vocabulary items defined by the Verifiable Credentials Working Group through different vocabularies, such as the <a href="https://www.w3.org/2018/credentials/">Verifiable Credentials Vocabulary</a>. Applications using additional, user-dependent vocabularies of their own are encouraged to create their own formal vocabularies, and corresponding JSON-LD context definitions, to ensure the interoperability of the credentials they produce and/or consume.
+            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> of a Verifiable Credential, 
+            as defined in [vc-data-model], provides a mapping from the standard terms used in credentials to the vocabulary 
+            items defined by the Verifiable Credentials Working Group through different vocabularies, 
+            such as the <a href="https://www.w3.org/2018/credentials/">Verifiable Credentials Vocabulary</a>. 
+            Applications using additional, issuer-dependent vocabularies of their own are encouraged to 
+            create their own formal vocabularies, and corresponding JSON-LD context definitions, 
+            to ensure the interoperability of the credentials they produce and/or consume.
         </p>
         <p>
-            There may be cases, however, when the creation of such formal vocabularies and context definitions is too much of a burden, such as for closed applications, experiments, or examples. The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> has been constructed in such a way that issuers may use new, hitherto undefined terms without further ado, and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/issuer-dependent#TERM</code>.
+            Not all document authors desire to define JSON-LD vocabularies or to make use of them with W3C Verifiable Credentials. 
+            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> has been constructed 
+            in such a way that issuers may use new, as-yet undefined terms without additional effort, 
+            and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/issuer-dependent#TERM</code>.
```

Comment:
```suggestion
            and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/issuer-dependent#TERM</code>.
            Authors that use this feature should be aware that consumers
            are expected to interpret the meaning of their terms relative to
            the issuer.

Reply:
I'm not sure where we want to indicate that the above can pose challenges, in the three party model, for digital wallet providers. They have to keep up with a list of issuers for which they understand the terms -- which will inevitably be kept small, containing only the largest VC issuers.

Reply:
I don't this we need to say more here, perhaps you can provide a reference to a standard that would be helpful for the reader to review regarding this? 

Like a section of another W3C TR? 

If this is a real issue, it must have been addressed in previous work, and we should be able to refer readers to a best practice.

Reply:
Explaining why there are two different options (why one choice might be made over the other) is important, especially when the lead in is: "Not all document authors desire to have to do X." I would expect readers to think "Oh, why would I do extra stuff if I don't have to?" The reason is that they are shifting the burden onto consumers and helping create market failures via concentrated power. We should at least say something about the impact on consumers if we aren't willing to be clear about the ultimate outcomes that are made more likely -- so readers understand why they would reuse existing vocabularies or collaborate to create new shared ones.

Reply:
>  perhaps you can provide a reference to a standard that would be helpful for the reader to review regarding this? Like a section of another W3C TR?

Hmm, well, one of the design goals for VCs was exactly to avoid this problem. The base issuer-dependent vocabulary re-introduces it to a certain degree. So, we created a spec to eliminate a problem that we are re-introducing in v2.0 (that you don't know what the issuer meant when they used a particular term, might never know, and you're guaranteed to conflict with others doing the same).

We should say something about the dangers of depending on issuer-dependent terms... the dangers were so bad that we had to create a new global standard to avoid the problem (Verifiable Credentials). We should speak to the danger if this default is used w/o thinking.

Reply:
@OR13,

> I don't this we need to say more here, perhaps you can provide a reference to a standard that would be helpful for the reader to review regarding this?
> 
> Like a section of another W3C TR?
> 
> If this is a real issue, it must have been addressed in previous work, and we should be able to refer readers to a best practice.

I should also say that following this argument prevents any new work from mentioning the problem -- which means no work will mention the problem. So -- we should mention the problem. Mentioning it will also mean people who have a similar conversation about this problem in the future on some other spec will have something in TR space to link to.

Reply:
I maybe misunderstood your original point then, I thought you were asserting there was a documented problem, that is caused by not using "issuer independent" contexts.

I was hoping to refer to that documentation be reference.

Are you saying we will be the first WG at W3C to describe this problem?

Reply:
@OR13,

> Are you saying we will be the first WG at W3C to describe this problem?

In context with the work we're doing here in a way people would readily understand -- I don't know the answer to your question. I know VCs were literally invented to help solve that problem. I'm happy for you to include a reference to another spec in TR space if you know of one, either in addition to the language I proposed, or if you find something that's very duplicative, we can just reference it.

You may be able to point to this generally as an informative reference around avoiding centralization: https://www.ietf.org/archive/id/draft-nottingham-avoiding-internet-centralization-09.html.